intelmq/bots/experts/rdap/expert.py

```diff
@@ -99,6 +99,13 @@ def process(self):
                             raise NotImplementedError("Authentication type %r (configured for service %r) is not implemented" % (service['auth'], domain_suffix))
 
                     resp = self.__session.get("{0}domain/{1}".format(service['url'], url))
+
+                    if resp.status_code < 200 or resp.status_code > 299:
+                        if resp.status_code == 404:
+                            return
+                        self.logger.debug("RDAP Server '%s' responded with '%d' for domain '%d'.", service['url'], resp.status_code, url)
```

Comment:
The domain is not an integer. That raised an error for me.
```suggestion
                        self.logger.debug("RDAP Server '%s' responded with '%d' for domain '%s'.", service['url'], resp.status_code, url)
```